src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -299,6 +300,21 @@ class RsErrorAnnotator : RsAnnotatorBase(), HighlightRangeExtension {
         error.addToHolder(holder)
     }
 
+    private fun checkUnstableAttribute(ref: RsReferenceElement, o: RsElement, holder: RsAnnotationHolder) {
+        val element = ref.reference.resolve() as? RsOuterAttributeOwner ?: return
+        for (outerAttr in element.outerAttrList) {
+            val metaItem = outerAttr.metaItem
+            if (metaItem.name != "unstable") continue
+            val args = metaItem.metaItemArgs ?: continue
+            for (innerMetaItem in args.metaItemList) {
+                if (innerMetaItem.name != "feature") continue
+                val value = innerMetaItem.litExpr?.stringValue ?: continue
+                val feature = CompilerFeature(value, FeatureState.ACTIVE)
+                feature.check(holder, o, "`${feature.name}`")
```

Comment:
`unstable` attribute frequently has `reason` value. Let's extract it to (don't forget that it's Rust string literal that can have escapes symbols) and use it as message